lib/textmessenger.rb

```diff
@@ -0,0 +1,17 @@
+require 'twilio-ruby'
+
+class TextMessenger
+
+  @client = Twilio::REST::Client.new(ENV['TWILIO_ACCOUNT_SID'], ENV['TWILIO_AUTH_TOKEN'])
```

Comment:
Metrics/LineLength: Line is too long. [89/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)